src/inmanta/protocol/methods.py

```diff
@@ -114,22 +121,22 @@ def create_environment(
     Create a new environment
 
     :param project_id: The id of the project this environment belongs to
-    :param name: The name of the environment
-    :param repository: The url (in git form) of the repository
-    :param branch: The name of the branch in the repository
-    :param environment_id: A unique environment id, if none an id is allocated by the server
+    :param name: The name of the environment.
+    :param repository: Optional. The URL (in git format) of the repository.
+    :param branch: Optional. The name of the branch in the repository.
+    :param environment_id: Optional. A unique environment id, if none an id is allocated by the server.
     """
 
 
 @method(path="/environment/<id>", operation="POST", client_types=[const.ClientType.api])
 def modify_environment(id: uuid.UUID, name: str, repository: str = None, branch: str = None):
     """
-    Modify the given environment
+    Modify the given environment.
 
-    :param id: The id of the environment
-    :param name: The name of the environment
-    :param repository: The url (in git form) of the repository
-    :param branch: The name of the branch in the repository
+    :param id: The id of the environment to modify.
+    :param name: The new name for the environment.
+    :param repository: Optional. The URL (in git format) of the repository.
+    :param branch: Optional. The name of the branch in the repository.
```

Comment:
We should mention here what the behavior is when branch or repository is set to None.